### PR TITLE
[frontend] fix remove cursor pointer on operator that is not clickable + Missing translations

### PR DESCRIPTION
**Pull request by jpkha**
### Proposed changes

* Remove cursor pointer on operator that is not clickable

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] I wrote test cases for the relevant uses case
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/FilterIconButtonGlobalOperator.tsx [hunk 1]
```diff
@@ -26,25 +26,6 @@ FilterIconButtonGlobalOperatorProps
   const { t } = useFormatter();
   const theme = useTheme();
   if (filtersUsedAsApiParameters.includes(displayedFilters[currentIndex].key)) {
-    if (
```
Comment (jpkha) [inline opencti-platform/opencti-front/src/components/FilterIconButtonGlobalOperator.tsx:29]: We won't have this scenario anymore

**Comment by Archidoit** [comment]
cursor pointer still present for the modes within a filter 
image


ok for the modes between different filters

**Comment by Archidoit** [comment]
everything is ok for pointers 👍

**Review by Archidoit — APPROVED** [review]

**Comment by Archidoit** [comment]
everything is ok for operators translation 👍

**Pull request merged**